src/main/java/mate/academy/controller/BookController.java

```diff
@@ -29,8 +29,8 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
-        return bookService.save(bookDto);
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
+        return bookService.save(requestDto);
```

Comment:
Don't provide BookDto in updateBook endpoint, create new dto or use CreateBookRequestDto
Also, add validation for this dto